crest/Assets/Crest/Crest/Scripts/OceanRenderer.cs

```diff
@@ -11,17 +12,54 @@ namespace Crest
     /// The main script for the ocean system. Attach this to a GameObject to create an ocean. This script initializes the various data types and systems
     /// and moves/scales the ocean based on the viewpoint. It also hosts a number of global settings that can be tweaked here.
     /// </summary>
+    [ExecuteAlways]
     public class OceanRenderer : MonoBehaviour
     {
         [Tooltip("The viewpoint which drives the ocean detail. Defaults to main camera."), SerializeField]
         Transform _viewpoint;
-        public Transform Viewpoint { get { return _viewpoint; } set { _viewpoint = value; } }
+        public Transform Viewpoint
+        {
+            get
+            {
+#if UNITY_EDITOR
+                // If scene view is focused, use its camera. This code is slightly ropey but seems to work ok enough.
+                if (EditorWindow.focusedWindow.titleContent.text == "Scene")
```

Comment:
`EditorWindow.focusedWindow && ` need to check that it exists

Reply:
Also, this will lose focus the moment I touch an inspector. We might need to store state.

Reply:
Good point! Attempt pushed now, seems to work now?

Reply:
Works great